jujugui/static/gui/src/app/components/shared/status/status/test-status.js

```diff
@@ -7,47 +7,59 @@ const enzyme = require('enzyme');
 const Status = require('./status');
 
 describe('Status', () => {
-  let entities;
+  let model, valueStore;
 
   const renderComponent = (options = {}) => enzyme.shallow(
     <Status
-      entities={options.entities || entities}
       generateApplicationOnClick={options._generateApplicationOnClick || sinon.stub()}
       generateApplicationURL={options._generateApplicationURL || sinon.stub()}
       generateCharmURL={options._generateCharmURL || sinon.stub()}
       generateMachineOnClick={options._generateMachineOnClick || sinon.stub()}
       generateMachineURL={options._generateMachineURL || sinon.stub()}
       generateUnitOnClick={options._generateUnitOnClick || sinon.stub()}
       generateUnitURL={options._generateUnitURL || sinon.stub()}
+      model={options.model || model}
       navigateToApplication={options._navigateToApplication || sinon.stub()}
       navigateToCharm={options._navigateToCharm || sinon.stub()}
-      navigateToMachine={options._navigateToMachine || sinon.stub()} />
+      navigateToMachine={options._navigateToMachine || sinon.stub()}
+      valueStore={options.valueStore || valueStore} />
   );
 
   beforeEach(() => {
-    entities = {
-      applications: [{
-        getAttrs: sinon.stub().returns({
-          status: {}
-        })
-      }],
-      machines: [{}],
-      model: {
-        cloud: 'aws',
-        environmentName: 'my-model',
-        modelUUID: 'myuuid',
-        region: 'neutral zone',
-        sla: 'advanced',
-        version: '2.42.47'
-      },
-      relations: [{}],
-      remoteApplications: [{}],
-      units: [{}]
+    model = {
+      cloud: 'aws',
+      environmentName: 'my-model',
+      modelUUID: 'myuuid',
+      region: 'neutral zone',
+      sla: 'advanced',
+      version: '2.42.47'
+    };
+    valueStore = {
```

Comment:
I wonder if there is value in having a general 'populate value store' method for these tests going forward to reduce the boilerplate on each new test.

Reply:
Yep, prob right.